src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/InstanceClient.cs

```diff
@@ -111,7 +111,7 @@ public async Task<List<Instance>> GetInstances(Dictionary<string, StringValues>
             if (response.StatusCode == HttpStatusCode.OK)
             {
                 string instanceData = await response.Content.ReadAsStringAsync();
-                List<Instance> instances = JsonConvert.DeserializeObject<List<Instance>>(instanceData);
+                List<Instance> instances = JsonConvert.DeserializeObject<QueryResponse<Instance>>(instanceData).Instances;
```

Comment:
@acn-sbuad  the changes for exisiting instance. What does that use

Reply:
Have restored previous version.